src/hooks/useGlobalState.tsx

```diff
@@ -20,7 +20,7 @@ export function withGlobalContext<P>(
   }
 }
 
-const useGlobalState = (): [GlobalState, React.Dispatch<AnyAction>] =>
+const useGlobalState = (): [globalState: GlobalState, dispatch: React.Dispatch<AnyAction>] =>
```

Comment:
Tuple types are named now?

Reply:
Can be, don't have to. Helps when destructing to know the names, like ,`state, setState`

Reply:
But when destructing tuples/arrays the name is what you give rather than what's defined on the type, no?
Or when you define this way you have to destruct by the given name?

Reply:
Nah, it's just in TS tooltip under that name instead of type only

Reply:
Nice!